framework/jobs/src/main/java/org/apache/cloudstack/framework/jobs/dao/AsyncJobDaoImpl.java

```diff
@@ -71,7 +72,7 @@ public AsyncJobDaoImpl() {
         expiringUnfinishedAsyncJobSearch.done();
 
         expiringCompletedAsyncJobSearch = createSearchBuilder();
-        expiringCompletedAsyncJobSearch.and("created", expiringCompletedAsyncJobSearch.entity().getCreated(), SearchCriteria.Op.LTEQ);
```

Comment:
This change is okay only if the 'created' it not used in any of the methods to query the result.

Reply:
I verified; it is only used by the async jobs garbage collector.